server/protocols/jmap-rfc-8621-integration-tests/distributed-jmap-rfc-8621-integration-tests/src/test/java/org/apache/james/jmap/rfc8621/distributed/DistributedAuthenticationTest.java

```diff
@@ -52,6 +52,5 @@ class DistributedAuthenticationTest implements AuthenticationContract {
         .extension(new AwsS3BlobStoreExtension())
         .server(configuration -> CassandraRabbitMQJamesServerMain.createServer(configuration)
             .overrideWith(new TestJMAPServerModule()))
-        .lifeCycle(JamesServerExtension.Lifecycle.PER_ENCLOSING_CLASS)
```

Comment:
I'm not keen on having modifications on other tests suites than Postgres tbh on a separate working branch than master

Reply:
I tend to agree with @Arsnael. We should not modify the contract test unless something is actually wrong with the contract test itself.

@hungphan227 maybe try more on releasing the connection upon James shutdown? As I think the production code deserves the connection release as well.
e.g. add this to the `SinglePostgresConnectionFactory`
```
    @PreDestroy
    public void dispose() {
        Mono.from(connection.close()).block();
    }
```

Reply:
I'm exploring an alternative at the moment as well.... doing like what we have with a part of our cassandra test, allowing a restart of the db after reaching a certain numbers of tests played

Reply:
@quantranhong1999 you commented on wrong thread

Reply:
> I'm not keen on having modifications on other tests suites than Postgres tbh on a separate working branch than master

I have reversed code and used a new solution